edx_exams/apps/lti/views.py

```diff
@@ -97,12 +96,39 @@ def start_proctoring(request, attempt_id):
     return redirect(get_lti_1p3_launch_start_url(launch_data))
 
 
-def end_assessment(request, attempt_id):  # pragma: no cover
+@api_view(['GET'])
+@require_http_methods(['GET'])
```

Comment:
Could you clarify why this would be a get? Because we're getting the attempt from the api but not actually directly accessing the model?

Reply:
Sure! This actually came up an earlier pull request of mine. You can see the discussion here: https://github.com/edx/edx-exams/pull/99#discussion_r1160141809. The gist is that this will be a request that the browser makes as part of the LTI flow, so it most appropriate as a GET. We had a discussion about handling the side-effect more elegantly but decided to leave it as is. I'm a little murky on this still, to be honest.

If this were a REST API call to update the attempt status, it would definitely be a `PUT`. What do you think?

Reply:
Nice! That makes perfect sense to me. Thank you for clarifying and for the context.